src/_help/enhance-documentation-content/topics.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Topics
+---
+
+- TOC
+{:toc}
+
+Topics are an exclusive custom property that we've created to allow you to add extended content to your API documentation. They enable you to add context, additional information, or even tutorial elements to guide your readers in using your API.
```

Comment:
Is this working both for OpenAPI and AsyncAPI?

Reply:
Yes it does 👍